<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="amount_found">
          <property name="text">
           <string>Found: 0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="found_table">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="gridStyle">
           <enum>Qt::NoPen</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="form_layout">
        <item>
         <layout class="QHBoxLayout" name="search_button_layout">
          <item>
           <widget class="QPushButton" name="new_scan">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>New scan</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="next_scan">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Next scan</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="undo_scan">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Undo scan</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="search_type">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>Exact value</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QComboBox" name="search_byte_size">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="currentIndex">
             <number>2</number>
            </property>
            <item>
             <property name="text">
              <string>1 Byte</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2 Bytes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4 Bytes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8 Bytes</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeView" name="saved_results"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAttach"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAttach">
   <property name="text">
    <string>Attach</string>
   </property>
   <property name="toolTip">
    <string>Attach to a process</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>new_scan</sender>
   <signal>clicked()</signal>
   <receiver>next_scan</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>475</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
